src/sentry/integrations/github/client.py

```diff
@@ -149,13 +150,14 @@ def get_token(self, force_refresh=False):
         return token
 
     def create_token(self):
+        headers = {
+            # TODO(jess): remove this whenever it's out of preview
+            "Accept": "application/vnd.github.machine-man-preview+json",
+        }
+        headers.update(jwt.authorization_header(self.get_jwt()))
```

Comment:
I think we should be copying this over to the `github_enterprise` client as well @flub

Reply:
They are slightly different because with GHE we store the domain with the installation id as the `integration.external_id` vs just the installation id as the `integration.external_id` for GH

Reply:
I just went ahead and updated it

Reply:
oh, thanks for already pushing this!  was just about to look at this.